I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 202, in script
    show Rogue_sprite handjob at sprite_location(0.2) as handjob
Exception: DynamicImage u'Rogue_handjob_over_hand_animation[action_speed]': could not find image. (u'Rogue_handjob_over_hand_animation3')

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 202, in script
    show Rogue_sprite handjob at sprite_location(0.2) as handjob
  File "C:\Users\Quique\Downloads\renpy-7.4.11-sdk/renpy/ast.py", line 1197, in execute
    show_imspec(self.imspec, atl=getattr(self, "atl", None))
  File "C:\Users\Quique\Downloads\renpy-7.4.11-sdk/renpy/ast.py", line 1167, in show_imspec
    atl=atl)
  File "C:\Users\Quique\Downloads\renpy-7.4.11-sdk/renpy/exports.py", line 768, in show
    sls.add(layer, img, key, zorder, behind, at_list=at_list, name=name, atl=atl, default_transform=default_transform, transient=transient)
  File "C:\Users\Quique\Downloads\renpy-7.4.11-sdk/renpy/display/core.py", line 1147, in add
    thing.visit_all(lambda d : None)
  File "C:\Users\Quique\Downloads\renpy-7.4.11-sdk/renpy/display/core.py", line 568, in visit_all
    d.visit_all(callback, seen)
  File "C:\Users\Quique\Downloads\renpy-7.4.11-sdk/renpy/display/core.py", line 568, in visit_all
    d.visit_all(callback, seen)
  File "C:\Users\Quique\Downloads\renpy-7.4.11-sdk/renpy/display/core.py", line 568, in visit_all
    d.visit_all(callback, seen)
  File "C:\Users\Quique\Downloads\renpy-7.4.11-sdk/renpy/display/core.py", line 568, in visit_all
    d.visit_all(callback, seen)
  File "C:\Users\Quique\Downloads\renpy-7.4.11-sdk/renpy/display/core.py", line 568, in visit_all
    d.visit_all(callback, seen)
  File "C:\Users\Quique\Downloads\renpy-7.4.11-sdk/renpy/display/core.py", line 558, in visit_all
    for d in self.visit():
  File "C:\Users\Quique\Downloads\renpy-7.4.11-sdk/renpy/display/image.py", line 744, in visit
    self.find_target()
  File "C:\Users\Quique\Downloads\renpy-7.4.11-sdk/renpy/display/image.py", line 652, in find_target
    raise Exception(error)
Exception: DynamicImage u'Rogue_handjob_over_hand_animation[action_speed]': could not find image. (u'Rogue_handjob_over_hand_animation3')

Windows-10-10.0.19041
Ren'Py 7.4.11.2266
Rogue-Like Chonky Mod 0.1f
Fri Jun 10 16:28:03 2022
